ci_framework/playbooks/ceph.yml

```diff
@@ -180,6 +180,14 @@
         name: cifmw_cephadm
         tasks_from: pools
 
+    - name: Deploy additional daemons
+      ansible.builtin.import_role:
+        name: cifmw_cephadm
+        tasks_from: rgw
+      vars:
+        cifmw_cephadm_rgw_network: 192.168.122.0/24
```

Comment:
I only hard-coded one network range variable and it had to be bound to its scope in the spec play.

https://github.com/openstack-k8s-operators/ci-framework/blob/main/ci_framework/playbooks/ceph.yml#L90

If the RGW network will be the same network, then a pre-task in the playbook could extract the network range from the ceph spec the same way that the first mon IP is set.

https://github.com/openstack-k8s-operators/ci-framework/blob/main/ci_framework/playbooks/ceph.yml#L153-L160

That way we could ensure we set the network in only once place and force consistency.

Reply:
Yeah, I'll push this additional change in my next iteration.